ts/sagas/backendInfo.ts

```diff
@@ -48,7 +49,12 @@ function* backendInfoWatcher(): IterableIterator<Effect> {
       // tslint:disable-next-line:saga-yield-return-type
       yield call(startTimer, BACKEND_INFO_LOAD_INTERVAL);
     } else {
-      yield put(backendInfoLoadFailure(new Error("Cannot read server info")));
+      const errorDescription = backendInfoResponse.isLeft()
```

Comment:
you can use [fold](https://github.com/gcanti/fp-ts/blob/1.18.0/src/Either.ts#L117) here:

```ts
const errorDescription = backendInfoResponse.fold(readableReport, ({status}) => `response status ${status}`);
```